src/cdef.rs

```diff
@@ -197,7 +197,7 @@ unsafe fn padding<BD: BitDepth>(
     for y in 0..h {
         let tmp = &mut tmp[(y + 2) * TMP_STRIDE..];
         let src = src + (y as isize * stride);
-        let src = &*src.data.slice::<BD, _>((src.offset.., ..x_end));
+        let src = &*src.data.slice::<BD, _>((src.offset.., ..x_end - 2));
```

Comment:
Is this change meant to be part of this PR? It seems unrelated to the rest of the changes.

Reply:
Whoops, no, I forgot to change the base branch.  That's #1178.